apps/desktop/src/types/biometric-message.ts

```diff
@@ -4,8 +4,9 @@ export enum BiometricAction {
 
   UnlockForUser = "unlockForUser",
   GetStatusForUser = "statusForUser",
-  SetKeyForUser = "setKeyForUser",
-  RemoveKeyForUser = "removeKeyForUser",
+  EnableBiometricUnlockForUser = "enableBiometricUnlockFroUser",
+  DisableBiometricUnlockForUser = "disableBiometricUnlockForUser",
+  ProvideBiometricUnlockKeyForUser = "provideBiometricUnlockKey",
```

Comment:
🎨 Consistency with the rest of biometric unlock's
```suggestion
  ProvideBiometricUnlockKeyForUser = "provideBiometricUnlockKeyForUser",
```